Makefile

```diff
@@ -0,0 +1,47 @@
+BINDIR=/usr/local/bin
+UNITDIR=/etc/systemd/system
+SYSCONFDIR=/usr/local/etc
+DEFAULTDIR=$(SYSCONFDIR)/default
+DESTDIR=
+PROGNAME=alertmanager_matrix
+
+all: bin/$(PROGNAME) $(PROGNAME).service
+
+ROOT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+
+.PHONY: clean dist rpm srpm install-unit install-prog install-default install
+
+clean:
+	cd $(ROOT_DIR) || exit $$? ; find -name '*.pyc' -o -name '*~' -print0 | xargs -0 rm -f
+	cd $(ROOT_DIR) || exit $$? ; rm -rf *.tar.gz *.rpm
+	cd $(ROOT_DIR) || exit $$? ; rm -rf bin $(PROGNAME).service
```

Comment:
I don't think the `cd` or the `find` should be necessary, which means this can be simplified to:

```
rm -rf *.tar.gz *.rpm
rm -rf bin $(PROGNAME).service
```

Reply:
That's only true if you run `make` from the current directory.  This isn't always the case, and this _protects you from data loss_ in case you do run it outside the current directory.